doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -4,13 +4,201 @@
+| 	      |                                                              |  1.10.1 |                         |
++             +                                                              +         +                         +
+| 	      |                                                              |  1.10.4 |                         |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|jasper       | https://github.com/mdadams/jasper                            | 1.900.1 | $JASPER_LIB             |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|zlib 	      | https://github.com/madler/zlib                               | 1.2.11  | $Z_LIB                  |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|libpng       | http://www.libpng.org/pub/png/libpng.html                    | 1.2.44  | $PNG_LIB                |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+
+
+Compiler:
+
+
++--------------------+------------------------------------------------------------------+
+|Compiler name       |    Branch name                                                   |
++====================+==================================================================+
+| Intel              |   16.0.3.210,16.3.210,18.0.1.163, 18.0.3.222,18.0.5.274, 19.0.2  |
++--------------------+------------------------------------------------------------------+
+|impi                |   2018.0.4, 2018.4.274,18.0.1, 2019.2.187                        |
++--------------------+------------------------------------------------------------------+
+
+
+These libraries are available on several platforms including Hera, Cheyenne and WCOSS. Installing these libraries on jet and gaea is a work in progress. Users who need to build the libraries. [https://github.com/NOAA-EMC/NCEPLIBS/wiki/Cloning-and-Compiling-NCEPLIBS]
+
+In addition to these libraries, other software is required such as a compiler (Intel or GNU), MPI library (impi), HDF5, cmake, Python, Perl and Ruby to compile the model and run regression tests.
+
+---------------------------------------------
+Making code changes using a forking workflow
+---------------------------------------------
+
+If developers would like to make code changes, they need to make a personal fork, set up upstream remote (for merging with the original ufs-weather-model), and create a branch for ufs-weather-model and each of the subcomponent repositories they want to change. They can then make code changes, perform testing and commit the changes to the branch in their personal fork. It is suggested that they merge their branch with the develop branch of the original repositories periodically to get the latest updates and bug fixes.
```

Comment:
This wording isn't clear - the last sentence - recommend changing "merge their branch", to "update their branch by merging the develop branch..."

Reply:
Done